rubocop.yml

```
@@ -439,6 +439,7 @@ Style/PerlBackrefs:
 
 Naming/PredicateName:
   Enabled: True
+  AllowedMethods: ['is_a', 'is_to_s']
```

Shouldn't this have been...
```suggestion
  AllowedMethods: ['is_a?', 'is_to_s']
```

---

Hmm, yea the default was with ?
I'll push another PR and fix it